controllers/users.js

```diff
@@ -284,6 +284,42 @@ class Users {
       }))
       .catch(err => next(err));
   }
+
+  /**
+   * get a list of authors
+   * @param {object} req
+   * @param {object} res
+   * @param {object} next
+   * @returns {object} res
+   */
+  static async getAllAuthors(req, res) {
+    const result = await User.findAll({ attributes: ['image', 'username', 'email'] });
```

Comment:
For the long-running operation like this of dealing with a database, it's better if you putt them in error handler block